lib/katello/engine.rb

```diff
@@ -135,9 +135,7 @@ class Engine < ::Rails::Engine
 
       # Lib Extensions
       ::Foreman::Renderer::Scope::Variables::Base.send :include, Katello::Concerns::RendererExtensions
-      if Katello.with_remote_execution?
-        ::ForemanRemoteExecution::Renderer::Scope::Input.send :include, Katello::Concerns::InputTemplateScopeExtensions
-      end
+      ::Foreman::Renderer::Scope::Base.send :include, Katello::Concerns::InputTemplateScopeExtensions
```

Comment:
Should this be available for provisoning templates and partitions tables as well?
If so, please also rename the concern to BaseTemplateScopr extensions.

Better would be to explicity include this for all the template types that need the errata macro. Bear in mind that we also habe report templates now that are not associated to a host.

Reply:
The goal was to expand it to work with all templates yes, (it was largely to make it easier to test, and there's not necessarily a reason to restrict it to just REX templates). 
Will rename it.